app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java

```diff
@@ -201,14 +201,14 @@ public static void setInfoRowDescription(InfoRowView view, String description) {
     //--------------------------------------------------------------
     // SearchResult
     //--------------------------------------------------------------
-    @BindingAdapter("searchResultIcon")
-    public static void setSearchResultIcon(TextView textView, @DrawableRes int iconResId) {
-        Drawable icon = ContextCompat.getDrawable(textView.getContext(), iconResId);
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
-            textView.setCompoundDrawablesRelativeWithIntrinsicBounds(icon, null, null, null);
-        } else {
-            textView.setCompoundDrawablesWithIntrinsicBounds(icon, null, null, null);
-        }
+    @BindingAdapter({"searchResultIcon", "mySession"})
+    public static void setSessionIcon(TextView textView, @DrawableRes int iconResId, boolean isMySession) {
```

Comment:
That's good! 👍 
Actually, I want to challenge this layout. Because designers often say "Put the icon behind the text!". 

Thank you for giving me the chance 😸